Added Materialize Page

FoxfordOH opened the pull request:
Used Materialized page to look at famous democratic presidents in history.

FoxfordOH committed:
Added Materialize Page

Used Materialized page to look at famous democratic presidents in history.

FoxfordOH committed:
Fixed File Names

FoxfordOH committed:
Fixed Headings

FoxfordOH committed:
Added Images Folder

FoxfordOH committed:
fixed folder name

FoxfordOH committed:
Adding Collapsible Initialization

FoxfordOH commented:
Having trouble getting Collapsible and Carousel to work. I customized the images and text and put the initialization code in my JavaScript file. I used the code from materialize, and it just doesn't want to work with me :,(

FoxfordOH committed:
Fixed Carousell and Collapsible

Thank you for the video; it helped a ton. I've been exhausted recently, so the help is incredibly appreciated.

AlyssaKirstine commented on interactions.js:
```diff
@@ -9,4 +9,31 @@ document.getElementById("mySubmit").onclick = function(){
+    const modalText = document.getElementById('modal-content-text');
+    modalText.innerText = `Thank you for subscribing! Updates will be sent to: ${email}`;
+    const modal = M.Modal.getInstance(document.getElementById('modal1'));
+    modal.open();
```
@FoxfordOH This code isn't working either, you'll also want to move it to the materialize page.

FoxfordOH replied:
I just tried to fix it and it is telling me modal is still not defined. I moved it to the materialize file.

FoxfordOH committed:
Attempted to Fix JS problem, keeps saying "modal not defined"

I attempted to fix this by moving some things around, but somehow the 'modal' variable is not getting defined.
